selfservice/strategy/password/validator.go

```diff
@@ -46,14 +48,17 @@ type DefaultPasswordValidator struct {
 
 	maxBreachesThreshold int64
 	ignoreNetworkErrors  bool
+
+	minIdentifierPasswordDist int
 }
 
 func NewDefaultPasswordValidatorStrategy() *DefaultPasswordValidator {
 	return &DefaultPasswordValidator{
-		c:                    http.DefaultClient,
-		maxBreachesThreshold: 0,
-		hashes:               map[string]int64{},
-		ignoreNetworkErrors:  true,
+		c:                         http.DefaultClient,
+		maxBreachesThreshold:      0,
+		hashes:                    map[string]int64{},
+		ignoreNetworkErrors:       true,
+		minIdentifierPasswordDist: 5,
```

Comment:
Can we make this configurable in the config file, and also document this in ? :)

Reply:
https://www.ory.sh/docs/kratos/self-service/strategies/username-email-password#security-and-defenses

Reply:
I think, this thing has been implemented. Right?

`func (s *DefaultPasswordValidator) Validate(ctx context.Context, identifier, password string) error {
	passwordPolicyConfig := s.reg.Config(ctx).PasswordPolicyConfig()
	if len(password) < int(passwordPolicyConfig.MinPasswordLength) {`

Reply:
You mean the configuration? Does not look like the values come from the config rn: https://github.com/ory/kratos/blob/10778fdd16a116b5dc8f4c2bdc96a895728d9aec/selfservice/strategy/password/validator.go#L155